ambry-frontend/src/test/java/com.github.ambry.frontend/AmbryBlobStorageServiceTest.java

```diff
@@ -232,6 +233,12 @@ public void badResponseHandlerAndRestRequestTest() throws Exception {
 
     responseHandler.reset();
     restResponseChannel = new MockRestResponseChannel();
+    ambryBlobStorageService.handlePut(restRequest, restResponseChannel);
```

Comment:
this does not belong in `badResponseHandlerAndRestRequestTest()` - we should have a different test like `AdminBlobStorageServiceTest::putFailureTest()`